vllm/distributed/device_communicators/shm_broadcast.py

```diff
@@ -428,7 +429,6 @@ def enqueue(self, obj):
 
     def dequeue(self):
         if self._is_local_reader:
-            overflow = False
```

Comment:
what's this change?

Reply:
oh sorry it's just unrelated nearby cleanup I saw while looking through that file, the line is unnecessary

Reply:
oh makes sense.